rcv/nes56_cv_manager.py

```diff
@@ -0,0 +1,150 @@
+import constants
+import cv2
+import logging
+import os
+import rcv_utils
+import socket
+import sys
+
+
+
+# globals
+API_structure = {
+        "front": False,
+        "found": False,
+        "p1": {"d": 0, "a": 0},
+        "p2": {"d": 0, "a": 0},
+    }
+
+
+class Nes56CvManager():
+    def __init__(self, conf):
+        logging.debug("entered Nes56CvManager.__init__(...)")
+        self._stop = False
+        self._conf = conf
+        self._client_socket = None
+        self._cap = None
+        self._width = None
+        self._height = None
+        self._video_source_type = None
+        self._video_source = None
+        self._camera_name = None
+        self._fps = None
+        self._roborio_ip = self._conf.get('rcv_server', 'roborio_ip')
+        self._roborio_port = int(self._conf.get('rcv_server', 'roborio_port'))
+        self._socket_timeout = int(self._conf.get('rcv_server', 'socket_timeout'))
+        self._show = bool(self._conf.get('rcv_server', 'show'))
```

Comment:
here you should change so when in the conf it is false the show will be false you should write if it equals to "False" or "True" because when converted to bool any string that isn't empty equals true

Reply:
If someone writes there something other than False it will show the image.
I don't see a problem with that